doc/release-process.md

```diff
@@ -141,29 +137,28 @@ git push  # Assuming you can push to the guix.sigs tree
 popd
 ```
 
-Codesigner only: Create Windows/macOS detached signatures:
-- Only one person handles codesigning. Everyone else should skip to the next step.
-- Only once the Windows/macOS builds each have 3 matching signatures may they be signed with their respective release keys.
+## Codesigning
 
-Codesigner only: Sign the macOS binary:
+### macOS codesigner only: Create detached macOS signatures
 
-    transfer bitcoin-osx-unsigned.tar.gz to macOS for signing
+    Assuming [signapple](https://github.com/achow101/signapple/) is installed and up to date with master branch.
```

Comment:
This line needs to be moved out of the bash block. Otherwise the link doesn't render:
![link](https://user-images.githubusercontent.com/863730/160077897-3e40fbc7-1efc-4599-a8fb-b52233c2b383.png)

Reply:
done